### PR TITLE
[CSL-1524] Remove obsolete stuff

### PR DESCRIPTION
**Pull request by georgeee**

### CODE REVIEW

File: lwallet/Pos/Wallet/Light/Web/Server.hs [hunk 1]
```diff
@@ -1,24 +0,0 @@
...
-module Pos.Wallet.Light.Web.Server
-       ( walletServeWebLite
-       ) where
-
-import           Universum
-
-import           Pos.Communication.Protocol (SendActions)
-import           Pos.Wallet.Light.Mode      (LightWalletMode)
-
-walletServeWebLite
```
Comment (gromakovsky) [inline lwallet/Pos/Wallet/Light/Web/Server.hs]: There is such code in `Main.hs`:
```
                Serve __webPort __webDaedalusDbPath -> error "light wallet server is disabled"
                -- Serve webPort webDaedalusDbPath -> worker walletServerOuts $ \sendActions ->
                --     walletServeWebLite sendActions webDaedalusDbPath False webPort
```

I think it makes sense to delete it (and `Serve` command) as well.

File: lwallet/Pos/Wallet/Light/Mode.hs [hunk 2]
```diff
@@ -122,16 +115,16 @@ instance HasCoreConstants => MonadSlots LightWalletMode where
...
+    getOwnUtxos addrs = filterUtxoByAddrs addrs <$> asks (unGenesisUtxo . lwcGenesisUtxo)
+    getBalance = getBalanceFromUtxo
 
 instance HasCoreConstants => MonadTxHistory LightWalletSscType LightWalletMode where
-    getBlockHistory = getBlockHistoryWallet
-    getLocalHistory = getLocalHistoryWallet
-    saveTx = saveTxWallet
+    getBlockHistory = error "getBlockHistory is not implemented for light wallet"
+    getLocalHistory = error "getLocalHistory is not implemented for light wallet"
+    saveTx = error "saveTx is not implemented for light wallet"
```
Comment (gromakovsky) [inline lwallet/Pos/Wallet/Light/Mode.hs]: Isn't it a breaking change? `saveTxWallet` was defined as `pure ()`.

Comment (georgeee) [inline lwallet/Pos/Wallet/Light/Mode.hs]: ok, will revert it (who knows if it's breaking)

**Comment by gromakovsky** [comment]
The Travis CI build failed

**Comment by adinapoli-iohk** [comment]
Build restarted.

File: lwallet/Pos/Wallet/Light/Mode.hs [hunk 3]
```diff
@@ -122,16 +115,16 @@ instance (HasCoreConstants, MonadSlotsData ctx LightWalletMode)
     currentTimeSlotting      = currentTimeSlottingSimple
 
 instance MonadGState LightWalletMode where
-    gsAdoptedBVData = gsAdoptedBVDataWallet
+    gsAdoptedBVData = pure Const.genesisBlockVersionData
 
 instance MonadBalances LightWalletMode where
-    getOwnUtxos = getOwnUtxosWallet
-    getBalance = getBalanceWallet
+    getOwnUtxos addrs = filterUtxoByAddrs addrs <$> asks (unGenesisUtxo . lwcGenesisUtxo)
```
Comment (gromakovsky) [inline lwallet/Pos/Wallet/Light/Mode.hs]: TIL it was always using genesis utxo. Weird :/

**Pull request merged**